src/celengine/solarsys.cpp

```diff
@@ -368,17 +363,15 @@ TimelinePhase* CreateTimelinePhase(Body* body,
         rotationModel = new ConstantOrientation(Quaterniond::Identity());
     }
 
-    TimelinePhase* phase = TimelinePhase::CreateTimelinePhase(universe,
+    auto phase = TimelinePhase::CreateTimelinePhase(universe,
                                                               body,
```

Comment:
incorrect indentation

Reply:
What do you mean?

Reply:
`body` should start just under `universe`